### ISSUE TITLE
Models to use

### ISSUE THREAD

**Issue by Lion-Mod** [janfreyberg/superintendent#63]
Jan love the idea of superintendent! I was thinking surely there is a way to efficiently label and in came active learning to save the day. Then in a jupyter environment.... love it!

My question is potentially a basic one but I would love to have an answer.

I'm looking to build a cnn image classifier. The data I have isn't labelled but I am willing to manually label some data hence superintendent. 

Why are we using LogisticRegression? Reading the docs I understand this is not the be all end all but is good.

So would it be easy to hotswap in say a CNN or efficientnet into the ClassLabeller?

**Comment by janfreyberg** [comment]
Hi,

Yes definitely - it should be a model that's more suited than Logistic Regression. I only put that in because it's a fast demo, and for MNIST performs very well.

If you want to use any other model, simply make sure it conforms with the scikit learn fit / predict API. For example, keras or skorch neural nets do this.

Hope that makes sense!
